src/object_properties.cpp

```diff
@@ -8,11 +8,24 @@
 #include "exceptions.h"
 #include "log.h"
 #include "util/serialize.h"
+#include "common/c_content.h"
 #include <sstream>
 #include <tuple>
 
 static const video::SColor NULL_BGCOLOR{0, 1, 1, 1};
 
+const struct EnumString es_ObjectVisual[] =
+{
+	{OBJECTVISUAL_UNKNOWN, "unknown"},
+	{OBJECTVISUAL_SPRITE, "sprite"},
+	{OBJECTVISUAL_UPRIGHT_SPRITE, "upright_sprite"},
+	{OBJECTVISUAL_CUBE, "cube"},
+	{OBJECTVISUAL_MESH, "mesh"},
+	{OBJECTVISUAL_ITEM, "item"},
+	{OBJECTVISUAL_WIELDITEM_ITEM, "wielditem_item"},
```

Comment:
Why not just "wielditem" / `OBJECTVISUAL_WIELDITEM`? Dropped items are invisible.